pmm-admin.go

```diff
@@ -261,6 +261,7 @@ It is supposed there could be only one instance of linux metrics being monitored
 However, you can add another one with the different name just for testing purposes using --force flag.
 
 [name] is an optional argument, by default it is set to the client name of this PMM client.
+[-- exporter_args] are additional exporter arguments.
```

Comment:
@borys-belinsky Maybe you know better way to describe `[-- exporter_args]`?

Reply:
Since it is mentioned in the syntax description that there is a special group of options called *exporter flags* there should be separate section for this group below. However, using the term *exporter flags* is confusing because the user does not (should not) interact with something called exporters. They use *monitoring services*. Therefore, I recommend that the syntax be changed to use this terminology: **pmm-admin add linux:metrics [-- monitoring-service-options] [command-options]**.
Another problem is that very often terms like *parameter*, *argument* (these two differ in code but are the same on the command line), *flag* (which should exclusively be used for on|off things, *option* are used interchangeably. I would suggest that we consistently use *option* as the most general and least technical. 
- Do not use *instance* (*monitoring service* is better): it makes no sense anywhere but deep in code.
- Do not play with dashes and underscores. I suggest that dashes be used consistently (like in pmm-admin).

## Optional
For the sake of consistency with the traditional usage of double dashes syntax , I would suggest that what you refer to as *-- exporter flags* appear after what you refer to as *flags*.